modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -87,7 +87,11 @@ def init_attributes(form)
         benefit_categories: map_benefit_categories(form['fieldBenefitCategories']),
         va_form_administration: form.dig('fieldVaFormAdministration', 'entity', 'entityLabel')
       }
-      mapped[:form_details_url] = "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}" if form['entityPublished']
+      mapped[:form_details_url] =
+        if form['entityPublished']
+          "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}"
+        else
+          '' end
```

Comment:
Just a style change here, the `end` should be on the next line 

```ruby
if form['entityPublished']
  "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}"
else
  ''
end
```

Reply:
I was thinking Rubocop forced me to do it that way, but I could have dreamed that up. Let me try it again...

Reply:
@rileyanderson Yeah, if I add a newline, rubocop says the method is too long. Do you have a preference how you want to get past that? If I put it all on one line, it looks too long to me

Reply:
Ah yeah it looks like rubocop is catching too many lines in the method 😞 . You could use a ternary operator here to keep method lines down and to keep styling consistent 

```ruby
mapped[:form_details_url] = form['entityPublished'] ? "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}" : ''
```

Reply:
done